src/utils/solanaHelpers.ts

```diff
@@ -319,35 +351,53 @@ export async function getEstimateBalanceChange(connection: Connection, tx: Trans
   }
 }
 
-export async function calculateTxFee(message: Message, connection: Connection): Promise<{ blockHash: string; fee: number; height: number }> {
+export async function calculateTxFee(
+  message: Message | VersionedMessage,
```

Comment:
should be accepting VersionedMessage type only rite.
as we use VersionedTransaction only